src/backend/el.d

```diff
@@ -65,22 +65,24 @@ struct elem
     enum IDelem = 0x4C45;   // 'EL'
     //elem_debug(e) assert((e)->id == IDelem)
 
+    version (OSX) // workaround https://issues.dlang.org/show_bug.cgi?id=16466
+        align(16) eve EV; // variants for each type of elem
+    else
+        eve EV;           // variants for each type of elem
+
```

Comment:
I put Srcpos at the end because it is rarely used - the others are also sorted by frequency of use. The overall size would remain the same if I used your sort order (absent the `debug short`), but it would likely be slower.